packages/code-snippet-block/src/settings.ts

```diff
@@ -0,0 +1,322 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
I had a compiling error: `Could not resolve entry module (src/index.tsx). (x2)`

Which is solved by renaming it to `index.tsx`. Or is there a reason why it is `*.ts`?

Reply:
You can add ```
        "serve": "frontify-cli block serve --entryPath src/index.ts",
        "deploy": "frontify-cli block deploy --entryPath src/index.ts",
``` to the `package.json` then it should work

Reply:
Every index.tsx file should be index.ts. @SamuelAlev, someone should fix this.